### PR TITLE
Updates OBS conversion to fully support RC v0.2

### PR DESCRIPTION
**Pull request by da1nerd**
Related to https://github.com/unfoldingWord-dev/door43.org/issues/366

This updates the OBS importer to use the rc0.2 spec.

### CODE REVIEW

File: converters/obs_converter.py [hunk 1]
```diff
@@ -25,9 +36,8 @@ def __init__(self, lang_code, git_repo, out_dir, quiet):
         self.git_repo = git_repo
         self.out_dir = out_dir
         self.quiet = quiet
-        # self.temp_dir = ''
 
-        if 'github' not in git_repo and 'file://' not in git_repo:
+        if 'github' not in git_repo and 'file://' not in git_repo and 'git.door43' not in git_repo:
```
Comment (phillip-hopper) [inline converters/obs_converter.py]: All dokuwiki repos are on github. What is the purpose of including 'git.door43' here?

File: converters/obs_converter.py [hunk 2]
```diff
@@ -71,16 +81,28 @@ def run(self):
         # download needed files from the repository
         files_to_download = []
         for i in range(1, 51):
-            files_to_download.append(str(i).zfill(2) + '.txt')
+            files_to_download.append(str(i).zfill(2) + '.md')
```
Comment (phillip-hopper) [inline converters/obs_converter.py]: All dokuwiki files end with .txt, not .md

**Review by phillip-hopper — CHANGES_REQUESTED** [review]
The class comment says this is for converting from Dokuwiki to RC, but the changes in the code don't appear to work with Dokuwiki. Please provide more explanation.

The conversion for OBS is not working. To run it, use this command:
```
python3 execute.py convert-obs --lang en --gitrepo https://github.com/Door43/d43-en --outdir ~/obs-en
```

**Comment by jag3773** [comment]
@phillip-hopper is Correct.  @neutrinog The source for the data should remain unchanged.

Phil's example is perfect, except don't run that for English!

**Comment by da1nerd** [comment]
ah ok. I couldn't find anything in github so I thought we were switching to gogs. Easy fix.

**Comment by da1nerd** [comment]
Ok switched back to just supporting dokuwiki from github and added some additional improvements to parsing titles and added some content tests.

**Comment by phillip-hopper** [comment]
@neutrinog Looks like there is an error in `test-setup.py`.

**Comment by jag3773** [comment]
Note that we have no rush here... changes are coming to reflect the RC 0.2 spec instead of 0.1.

**Comment by da1nerd** [comment]
This is ready for another look but I can't figure out why coverage keeps failing.

File: converters/obs_converter.py (hunk 1 above)
Comment (da1nerd) [reply converters/obs_converter.py]: this has been fixed.

File: converters/obs_converter.py (hunk 2 above)
Comment (da1nerd) [reply converters/obs_converter.py]: this has been fixed.

**Pull request merged**